PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -13,6 +13,13 @@
 /// \brief Reconstruction of Omegac0 and Xic0 decays candidates
 /// \author Federica Zanone <federica.zanone@cern.ch>, Heidelberg University
```

Comment:
Please add yourself on the list of authors with a second `/// \author` line.

Reply:
Done.